### DISCUSSION TITLE
Telepítési hiba

### DISCUSSION THREAD

**Discussion by azsak79** [amaximus/met_alerts_hu#8] [General]
Folyamatosan ezeket a  hibákat kapom:

Component error: region_id - Integration 'region_id' not found.
Component error: name - Integration 'name' not found.
Component error: platform - Integration 'platform' not found.
Component error: county_id - Integration 'county_id' not found.

Van ötleted mi lehet a gond? HASS alól telepítettem az addont.

**Comment by amaximus** [comment]
Be tudnád ide másolni a szenzor definicióját?

**Reply by azsak79** [reply]
platform: met_alerts_hu
name: 'MET alerts'
region_id: 106
county_id: 706

**Reply by amaximus** [reply]
gyanítom külön file-ban van.
Be van kapcsolva valamilyen debug level?
Ide tudnád másolni a log file-ból azt a pár sort ami erre szenzorra hibaként megjelenik?

Milyen HA installáció?
Hányas HA verzió?
Hányas MET Alerts Hungary verzió?

**Reply by azsak79** [reply]
core-2021.6.6
supervisor-2021.06.8
 MET Alerts a legfrissebb ami ma jött ki.

Semmilyen log bejegyzés nincs ezzel kapcsolatban. Akkor kapom ezt az üzenetet ha a konfig ellenőrzésre megyek.

**Reply by amaximus** [reply]
A konfig ellenőrzés alatt a `hass --script check_config -c <conf_dir>` érted vagy valami mást?

**Reply by azsak79** [reply]
Konfiguráció - Konfiguráció ellenőrzése menüpont

**Reply by amaximus** [reply]
Valami azzal lehet ahogyan beírtad azt a pár sort. Nálam (supervisort leszámítva) az van ami nálad, a konfig check lefut szó nélkül. Igaz én yaml módban használom a HAt, így terminálban írom a yaml file-t, nem pedig az UI-n keresztül. Szerintem ott nézd meg ahol beírtad a szenzor definiciót.

**Reply by azsak79** [reply]
Logger: homeassistant.setup
Source: setup.py:172
First occurred: 19:33:21 (2 occurrences)
Last logged: 19:33:28

Setup failed for met_alerts_hu: No setup or config entry setup function defined.
Setup failed for default_config: Could not set up all dependencies.

**Reply by amaximus** [reply]
Maga az integráció nem támogatja a config_flow-t így kézzel kell létrehozni a szenzor definiciót. Nem tudom hogy vannak nálad a kézi-definiálású szenzorok definiálva, de ahol/ahogy vannak, úgy hozd létre a met_alerts_hu-s szenzorét is, utána meg HA restart.
